hack/update-capi-crds.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# find the module version and the location on the system
+src=$(go mod download -json sigs.k8s.io/cluster-api  | jq -r .Dir)
+
+# -p will not fail if the dir exists
+mkdir -p capi-config
```

Comment:
what about using `config-capi` instead so we can get all the folders containing config files close together  ...

Reply:
yes! that is a great idea.